src/cloudai/schema/system/slurm/slurm_system.py

```diff
@@ -567,10 +560,8 @@ def parse_squeue_output(self, squeue_output: str) -> Dict[str, str]:
 
                 node_list_part, user = parts[0], "|".join(parts[1:])
                 # Handle cases where multiple node groups or ranges are specified
-                node_groups = node_list_part.split(",")
-                for node_group in node_groups:
-                    # Process each node or range using parse_node_list
-                    for node in self.parse_node_list([node_group.strip()]):
+                if node_list_part:
+                    for node in self.parse_node_list(node_list_part.strip()):
```

Comment:
IMO `parse_node_list` should correctly handle empty output and return an empty list.